ui/k-info-panel/switch_info.kytos

```diff
@@ -61,7 +71,60 @@
        window.d3.json(endpoint, function(error, result) {
          self.flows = result[dpid].flows
        })
-     }
+     },
+     bt_add_metadata() {
+       var _this = this
+       let request = $.ajax({
+                        type: "POST",
+                        url: this.$kytos_server_api + "kytos/topology/v3/switches/" + this.metadata.dpid
+                              + "/metadata",
+                        async: true,
+                        data: this.to_add,
+                        dataType: "json",
+                        contentType: "application/json; charset=utf-8",
+       });
+       request.done(function() {
+         let notification = {
+              icon: 'gear',
+              title: 'Add metadata: Success',
+              description: '"' + _this.to_add + '" was added to the metadata. Switch: ' + _this.metadata.id,
+         }
```

Comment:
@Alopalao could you please reset the `to_add` value to its default value when the request is successful? Also same for `to_delete`. It turns out that on other components that we use, once they succeed they're typically reset in case the operator wants to keep adding or deleting new ones.

Reply:
Updated to specifications on commit [c8bcaa6f7b6d9aa520961067087775f287fd4df7](https://github.com/kytos-ng/topology/pull/74/commits/c8bcaa6f7b6d9aa520961067087775f287fd4df7)